Libraries/Microsoft.Teams.Common/Http/HttpClient.cs

```diff
@@ -89,10 +89,11 @@ protected HttpRequestMessage CreateRequest(IHttpRequest request)
             }
             else
             {
-                httpRequest.Content = JsonContent.Create(request.Body, options: new JsonSerializerOptions()
+                string body = JsonSerializer.Serialize(request.Body, new JsonSerializerOptions()
```

Comment:
Looks like ABS requires an explicit UTF8 encoding (or something like that), and I was not able to make it work with `JsonContent`, changing to `StringContent` made the trick

Reply:
Creating a new `JsonSerializerOptions` will cause performance overheards since it generates and caches metadata internally. The [guidance](https://learn.microsoft.com/en-us/dotnet/standard/serialization/system-text-json/configure-options#reuse-jsonserializeroptions-instances) is to reuse the same instance (it's thread-safe btw)